### DISCUSSION TITLE
Issue with definitions of ‘Waveform’ and ‘Wavelength’

### DISCUSSION THREAD

**Discussion by gregfowlerphd** [CommonCoreOntology/CommonCoreOntologies#345] [Announcements]
The definitions read as follows:

> **Waveform:** A Wave Process Profile that is the shape of the Wave Cycles of the Wave Process when depicted in a visual graph.

> **Wavelength:** A Wave Process Profile that is the distance the Wave Process traverses during one Wave Cycle.

However, it would seem that neither of these definitions can be correct. A WPP is an occurrent, whereas I take it that a shape is a quality (which is a subclass of continuant). Similarly, a distance would also seem to be a non-occurrent. (Note that this latter point also applies wrt the definition for sound wavelength.)

I admit that, though I’ve looked a bit into the phenomena (and will continue to do so), I’m not sure how to revise the definitions. So I thought I’d post about the problem in case anyone else has any insights.

**Comment by oliviahobai** [comment]
Moving to discussion so physicists can contribute
